valhalla/midgard/gridded_data.h

```diff
@@ -115,55 +115,77 @@ template <std::size_t dimensions_t> class GriddedData : public Tiles<PointLL> {
     // function stored in the array
     int case_table[3][3][3] = {
         {{0, 0, 8}, {0, 2, 5}, {7, 6, 9}},
-        {{0, 3, 4}, {1, 3, 1}, {4, 3, 0}},
+        {{0, 3, 4}, {1, 0, 1}, {4, 3, 0}},
```

Comment:
this is the case when all three vertexes lie on the contour level(so sh[] == 0). As segment orientation is introduced, it's unclear how to orient the segment in this case. It looks safe just to omit such case(when all sh[] == 0) so i did it.

The picture from the original source also relates this case to the "do-nothing" case. (see "g" case) 
http://paulbourke.net/papers/conrec/
![conrec3](https://user-images.githubusercontent.com/14297128/110773139-791e3100-826d-11eb-82f5-8397b77d2b16.gif)